.github/append_manifests.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
can maybe replaced by `yq` here? I'm a bit.... :D  reluctant to add python as a dependency

Reply:
see my comment below :D

Reply:
![](https://media.giphy.com/media/3o72FauRvj76qp5bI4/source.gif)